src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -621,6 +665,16 @@ private async Task<bool> CanUpdateDataFiles(Guid releaseVersionId)
                 return ValidationActionResult(CannotRemoveDataFilesOnceReleaseApproved);
             }
 
+            if (releaseFile.PublicApiDataSetId is not null)
+            {
+                return Common.Validators.ValidationUtils.ValidationResult(new ErrorViewModel
+                {
+                    Code = ValidationMessages.CannotDeleteApiDataSetReleaseFile.Code,
+                    Message = ValidationMessages.CannotDeleteApiDataSetReleaseFile.Message,
```

Comment:
I originally had this, but then thought that technically it's a combination of the `releaseVersionId` and `fileId` that causes the error, not the `fileId` alone. So thought it might be a bit misleading

Reply:
Discussed, and agreed to remove the `Path`